apps/sensenet/src/components/AddButton.tsx

```diff
@@ -139,28 +145,37 @@ export const AddButton: React.FunctionComponent<AddButtonProps> = props => {
             justifyContent: 'space-evenly',
           }}
           button={true}
-          onClick={() => setShowSelectType(true)}
+          onClick={(event: React.MouseEvent<HTMLElement, MouseEvent>) => {
+            setAnchorEl(event.currentTarget)
+            setShowSelectType(true)
+          }}
           disabled={!isAvailable}>
           <ListItemIcon>
             <Add />
           </ListItemIcon>
           <ListItemText primary={localization.addNew} />
         </ListItem>
       )}
-      <SwipeableDrawer
-        anchor="bottom"
-        onClose={() => setShowSelectType(false)}
-        onOpen={() => {
-          /** */
+      <Popover
+        style={{ maxHeight: '85%' }}
```

Comment:
We should set a min and max width for this element. @CsabaPeter please help us out here.